tests/run_pytest

```diff
@@ -2,10 +2,32 @@
 
 SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &>/dev/null && pwd)
 
+RUN_FOLDER=/var/tmp/tdxtest/
+
 install_deps() {
-		sudo apt install -y python3-pytest
+  sudo apt install -y python3-pip python3-venv
 }
 
+setup_venv() {
+  deactivate &> /dev/null || true
+  python3 -m venv ${RUN_FOLDER}/venv
+  source ${RUN_FOLDER}/venv/bin/activate
+  python3 -m pip install paramiko \
```

Comment:
I think we should freeze the python package versions, put them in a `requirements.txt` file, and install from that file. That way we're less prone to errors occurring in the future due to package upgrades (pip will just install the latest version of a package by default, so six months from now the versions of these packages will look quite different).